lib/src/simple_op_heads_store.rs

```diff
@@ -88,16 +88,16 @@ impl OpHeadsStore for SimpleOpHeadsStore {
         }
     }
 
-    fn get_op_heads(&self) -> Vec<OperationId> {
+    fn get_op_heads(&self) -> OpHeadStoreResult<Vec<OperationId>> {
         let mut op_heads = vec![];
-        for op_head_entry in std::fs::read_dir(&self.dir).unwrap() {
+        for op_head_entry in std::fs::read_dir(&self.dir).map_err(io_to_read_error)? {
```

Comment:
Perhaps, we can use `PathError` to attach `.context(&self.dir)`. Bare `io::Error` doesn't give much information.